### PR TITLE
added touch sensors to the feet of NUgus.proto

### PR DESCRIPTION
**Pull request by NDxiura**

### CODE REVIEW

**Review by ysims — COMMENTED** [review]
Ah some copy and paste fun 😅  - check out the 'files changed' tab and revert the removed stuff. Might be easiest to copy the two touch sensor blocks of code somewhere, copy back in the current `main` version of `NUgus.proto`, and then copy back in the two touch sensor blocks.

**Comment by moosahassanx** [comment]
Bit hard to follow without any comments.
